src/assets/toolkit/styles/sandbox/sil-pwa.css

```diff
@@ -0,0 +1,74 @@
+@import "../base/_config.css";
+
+/* Overrides Sky > a { color: white} */
+.u-linkBlue {
+  color: var(--color-blue) !important;
+}
+
+.u-linkBlue:--enter {
+  color: var(--color-blue) !important;
+  opacity: 0.75 !important;
+}
+
+/* Permalink color */
+.u-linkGrayDark {
+  color: color(var(--color-gray) l(-35%) s(+5%)) !important;
+}
+
+.u-linkGrayDark:--enter {
+  color: color(var(--color-gray) l(-35%) s(+5%)) !important;
+  opacity: 0.75 !important;
+}
+
+/* Permalink bg */
+.u-bgGrayLight {
+  background-color: color(var(--color-gray) l(+5%)) !important;
+}
+
+/* Used for the date on the post page */
+.u-textGrayDark {
+  color: color(var(--color-gray) l(-45%) s(+5%)) !important;
+}
+
+/* Setting columns with min-width
+ * Probably break gap size into another utility
+ */
+.u-columns {
+  columns: 14.5rem 5;
+  column-gap: calc(1em * var(--ms1));
+}
```

Comment:
FYI, for the final styles I might opt to use `display: grid` and a simplified fallback.